src/components/HomePage.js

```diff
@@ -0,0 +1,15 @@
+import Countries from './Countries';
+
+const HomePage = () => (
+  <>
+    <div className="container">
+      <div className="headline">
+        <h2>CORONA VIRUS STATISTICS AROUND THE WORLD</h2>
+      </div>
+      <Countries />
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, " `When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.` ", please kindly add a filtering component to your home page(for a example a search). Checkout this [article](https://blog.logrocket.com/create-search-bar-react-from-scratch/) for inspiration.

Reply:
@t-yanick Hello thank you for the reviews you gave so much clarification that I was not aware of before, I have addressed all the issues mentioned in this review, but I would like to express my concern about recording the loom video, my computer has a temporary webcam problem so I could not record the video showing both screen and my face at the same time as it is in the project requirements, I hope you can bear with me while I am trying my best to fix the issue. thanks